src/usr/local/www/services_ntpd_acts.php

```diff
@@ -0,0 +1,317 @@
+<?php
```

Comment:
This file doesn't seem to belong to this PR

Reply:
all fixed